load.go

```diff
@@ -445,6 +445,13 @@ func (s *Spec) FillDefaults() {
 			s.Patches[k][i].Strip = &strip
 		}
 	}
+
+	for k, ac := range s.Artifacts.Libexec {
```

Comment:
We're not doing this for any of the other artifact types, I think it would be good to leave this up to the underlying implementation to deal with rather than changing it in the spec.

Reply:
Makes sense

Reply:
done